src/components/DatetimePicker/DatetimePicker.vue

```diff
@@ -0,0 +1,78 @@
+<!--
+  - @copyright Copyright (c) 2018 John Molakvoæ <skjnldsv@protonmail.com>
+  -
+  - @author John Molakvoæ <skjnldsv@protonmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<datepicker
+		v-bind="$attrs"
+		:minute-step="10"
+		:lang="lang"
+		:clearable="false"
+		:first-day-of-week="firstDay"
+		v-on="$listeners" />
+</template>
+
+<script>
+import Datepicker from 'vue2-datepicker'
+
+/**
+ * hijack the display function and avoid the
+ * top and left original positionning
+ */
+Datepicker.methods.displayPopup = function() {
+	const popupElmt = this.$el.querySelector('.mx-datepicker-popup')
+	if (popupElmt && !popupElmt.classList.contains('popovermenu')) {
+		popupElmt.className += ' popovermenu menu-center open'
+	}
+}
+
+export default {
+	name: 'DatetimePicker',
+
+	components: {
+		Datepicker
+	},
+
+	inheritAttrs: false,
+
+	computed: {
+		firstDay() {
+			return window.firstDay
```

Comment:
who sets this global?

Reply:
Nextcloud :)

Reply:
How ugly :see_no_evil:

Can we at least externalize this to a helper/util module to hide this ugliness and make it more obvious where this is coming from?

Reply:
:cry: 
How would you do it?  :)